### PR TITLE
provide option to use arxiv id as bibtex key

### PR DESCRIPTION
**Pull request by pseyfert**
I guess it's rather self explanatory… I want to do `\cite{1507.03414}`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,3 +57,6 @@ you are good to go.
 
 ## Dependencies
 `arXbib` is written in Python3.4 and needs the [beautifulsoup](http://www.crummy.com/software/BeautifulSoup/bs4/doc/) parser.
+
+[inspire]:    https://inspirehep.net/
```
Comment (seneubert) [inline README.md:61]: why don't you like the links up there?

**Comment by seneubert** [comment]
Will merge because the links issue in the README is no show stopper.

File: README.md (hunk 1 above)
Comment (pseyfert) [inline README.md:61]: don't like to have link-duplication. and conda was just on the same visible page … but really no strong opinion do as you wish

Comment (seneubert) [inline README.md:61]: Ok fine with me.

Thanks for the PR! Good idea!

**Pull request merged**